src/pages/round/[...number].tsx

```diff
@@ -75,57 +64,80 @@ export default function Round() {
     }
   }, [display, roundList, number]);
 
-  const Url = "/api/v1/bribedata/" + requestedRound;
-
   console.log(roundList.latest);
   console.log(number[0], requestedRound);
-
-  const { data: bribedata, isLoading } = useQuery(["bribedata", Url], () =>
-    fetchRoundData(Url)
-  );
-
-  if (isLoading) {
-    return <Center>Loading ... </Center>;
-  }
+  console.log(bribeData?.header);
+  console.log(bribeData?.bribelist);
 
   return (
     <>
-      <Center>
-        <Box>
-          <pre>
-            Round {requestedRound} -- <Link href={Url}>{Url}</Link>
-          </pre>
-        </Box>
-      </Center>
-      <Center>
-        <Box>
-          <pre>{bribedata?.version}</pre>
-        </Box>
-      </Center>
+      <Box>
+        <Center>
+          <Text fontSize="4xl">{bribeData?.header.roundName}</Text>
+        </Center>
+        <Center>
+          <Text fontSize="1.4rem">
```

Comment:
This is good for now, but on the todo list should be bake this style into the theme